common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -277,6 +276,28 @@ def setUpClass(cls):
         cls.store = modulestore()
 
     @classmethod
+    @contextmanager
+    def setUpClassAndTestData(cls):
+        """
+        For use when the test class has a setUpTestData() method that uses variables
+        that are setup during setUpClass() of the same test class.
+        """
```

Comment:
It might be worth including a short usage example in this docstring. I've found that to be very helpful when working with this class in the past.